dockstore-webservice/src/test/resources/json.cwl

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Hmmm, json cwl files. Does this mean we get https://github.com/dockstore/dockstore/issues/3658 for free?

Reply:
It's only been tested lightly, but I think the answer is "yes".